webpack.config.js

```diff
@@ -1,41 +1,44 @@
-const webpackConfig = require('terra-dev-site/src/config/webpack.config');
-const I18nAggregatorPlugin = require('terra-i18n-plugin');
-const i18nSupportedLocales = require('terra-i18n/lib/i18nSupportedLocales');
+const defaultWebpackConfig = require('terra-dev-site/config/webpack/webpack.config');
 const path = require('path');
-const glob = require('glob');
 
 
-const searchPattern = path.resolve(process.cwd(), 'node_modules', 'terra-*', 'packages', '*');
-
-const translationsDirectories = glob.sync(searchPattern);
-
-const babelLoaderRule = {
-  test: /\.(jsx|js)$/,
-  exclude: /node_modules(?!\/terra-)/,
-  use: 'babel-loader',
+const webpackConfig = (env, argv) => {
+  const config = defaultWebpackConfig(env, argv);
+  const babelLoaderRule = {
```

Comment:
Was this rule still needed? I know at were looking into dropping it in toolkit

Reply:
Yes, it's still needed until we are running babel on our released example documentation.